docs/usage/getting-started/running.md

```diff
@@ -18,6 +18,26 @@ Self-hosting Renovate means that you are the "administrator" of the bot, which e
 
 ### Available distributions
 
+#### Forking Renovate app
+
+"Forking Renovate" is the sister app to Renovate Bot.
+The difference is that Forking Renovate operates using forks of repositories instead of as a collaborator, so needs only read-only permissions rather than read-write.
+It works on public (e.g. open source) repositories only.
+
+[Install Forking Renovate from GitHub App](https://github.com/apps/forking-renovate).
+
+##### Benefits
+
+Forking Renovate needs only `read` level access to any repository it runs on.
+
+##### Drawbacks
+
+If you use Forking Renovate, you'll miss out on these features of the regular Renovate app:
+
+- Dependency Dashboard (probably needs `write` rights to be able to open the issue?)
+- Automerge with the `branch` automerge strategy (needs `write` rights to create a branch in the first place)
+- List more big ticket items here
```

Comment:
Please check if I'm correct in listing the Dependency Dashboard and automerge with `branch` strategy as things you'll miss out on if you use the Forking Renovate app.

We probably want to nudge/steer our users towards the full-fat app if possible?